content/blog/announce-tonic-0-1.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Tonic: gRPC has come to async/await!"
+url: "/tonic-grpc-has-come-to-async-await"
+date: 2019-10-01T10:00:00-04:00
+draft: false
+---
+
+# Tonic: gRPC has come to async/await!
+
+I am pleased to finally announce a crate that I have been working very hard on for the
+past few months. [`tonic`] has finally hit the initial [`0.1.0-alpha.1`] release! Stable
+releases will follow in the coming months.
+
+### What is Tonic?
+
+> Tonic is a gRPC over HTTP/2 implementation focused on _**high performance**_, _**interoperability**_, and _**flexibility**_. This library was created to have first class support of _async/await_ and to act as a core building block for production systems written in Rust.
+
+Tonic's previous iteration was called [`tower-grpc`] and was built for the need to have a production
+ready gRPC implementation for [linkerd]'s proxy. This spawned the creation of the [`tower`] library
+which is now being used in production in both [`linkerd-proxy`] and [`vector`]. As well as being used in
```

Comment:
"As well as being used in research systems such as the [`noria`]database developed at MIT" shouldn't be its own sentence. I'd make take the existing users a list instead.